src/core/atomic.d

```diff
@@ -90,7 +90,7 @@ T atomicLoad(MemoryOrder ms = MemoryOrder.seq, T)(ref shared const T val) pure n
 }
 
 /// Ditto
-TailShared!T atomicLoad(MemoryOrder ms = MemoryOrder.seq, T)(ref shared const T val) pure nothrow @nogc @trusted
```

Comment:
Wait a second, this is not right. `ref` should not be removed.

Reply:
I remember `ref return` not compiling, I think it should be `ref return scope`. I removed this change, will look at it later today (or in a follow-up PR).